data/os/RedHat/CentOS.yaml

```diff
@@ -1,47 +1 @@
----
-yum::os_default_repos:
-  - 'extras'
-  - 'centosplus'
-  - 'fasttrack'
-  - 'extras-source'
-  - 'base-debuginfo'
-yum::repos:
-  extras:
-    enabled: false
-    descr: 'CentOS-$releasever - Extras'
-    mirrorlist: 'http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=extras&infra=$infra'
-    gpgcheck: true
-    gpgkey: "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-%{facts.os.release.major}"
-    target: '/etc/yum.repos.d/CentOS-Base.repo'
-
-  centosplus:
-    enabled: false
-    descr: 'CentOS-$releasever - Plus'
-    mirrorlist: 'http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=centosplus&infra=$infra'
-    gpgcheck: true
-    gpgkey: "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-%{facts.os.release.major}"
-    target: '/etc/yum.repos.d/CentOS-Base.repo'
-
-  fasttrack:
-    enabled: false
-    descr: 'CentOS-$releasever - fasttrack'
-    mirrorlist: 'http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=fasttrack&infra=$infra'
-    gpgcheck: true
-    gpgkey: "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-%{facts.os.release.major}"
-    target: '/etc/yum.repos.d/CentOS-fasttrack.repo'
-
-  extras-source:
-    enabled: false
-    descr: 'CentOS-$releasever - Extras Sources'
-    baseurl: 'http://vault.centos.org/centos/$releasever/extras/Source/'
-    gpgcheck: true
-    gpgkey: "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-%{facts.os.release.major}"
-    target: '/etc/yum.repos.d/CentOS-Sources.repo'
-
-  base-debuginfo:
-    enabled: false
-    descr: 'CentOS-$releasever - Debuginfo'
-    baseurl: 'http://debuginfo.centos.org/$releasever/$basearch/'
-    gpgcheck: true
-    gpgkey: "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Debug-%{facts.os.release.major}"
-    target: '/etc/yum.repos.d/CentOS-Debuginfo.repo'
+--- {}
```

Comment:
I prefer it to delete the file, but I don't think there is a best practice for it.